wagtail/contrib/redirects/management/commands/import_redirects.py

```diff
@@ -0,0 +1,182 @@
+import os
+
+import tablib
+from django.core.management.base import BaseCommand
+from tablib.formats import registry
+from wagtail.contrib.redirects.forms import RedirectForm
+from wagtail.core.models import Site
+
+
+class Command(BaseCommand):
+    help = "Imports redirects from .csv, .xls, .xlsx"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--src", help="Path to file", type=str, required=True,
+        )
+        parser.add_argument(
+            "--site_id", help="The site where redirects will be associated", type=int,
+        )
+        parser.add_argument(
+            "--permanent",
+            help="Save redirects as permanent redirects",
+            type=bool,
+            default=True,
+        )
+        parser.add_argument(
+            "--from_index",
+            help="The column where to read from link",
+            default=0,
+            type=int,
+        )
+        parser.add_argument(
+            "--to_index", help="The column where to read to link", default=1, type=int,
+        )
+        parser.add_argument(
+            "--dry_run",
+            default=False,
+            help="Run only in test mode, will not create redirects",
+            type=bool,
+        )
+        parser.add_argument(
+            "--ask", help="Ask before creating", default=False, type=bool,
+        )
+
+        available_formats = [key for key in registry._formats]
+        parser.add_argument(
+            "--format",
+            help="Source file format (.csv, .xls etc)",
+            choices=available_formats,
+            type=str,
+        )
+
+        parser.add_argument(
+            "--offset", help="Import starting with index", type=int, default=-1
+        )
+        parser.add_argument(
+            "--limit", help="Limit import to num items", type=int, default=-1
```

Comment:
the default here might be confusing, if the arg is not defined, does that mean that only the first row will be included.

Maybe the default could be `None` and if `None`, all lines will be imported.

Reply:
I usually use `-1` as a replacement to `None` when dealing with positive ints, mainly to improve type integrity. So -1 in this case will import all rows. But in this case I think you are right, since -1, in the context of value operations in python could mean we want to import the last row. I'm changing to None.